### ISSUE TITLE
Drone event system

### ISSUE THREAD

**Issue by jamrizzi** [drone/drone#2311]
I have a docker container I need to spin up in parallel after it is built. I can't do this using groups because the container runs indefinitely, and the group gets stuck. Yes, I can use a timeout to force shutdown the container after a certain amount of time, but it is a hacky solution and forces me to override the default command inside the container.

There are two possible ways to accomplish this, but they would both be dependant on some sort of primitive event system.

1. Services could be dependant on a certain step in the pipeline.

2. Drone groups could shut down when a certain container in the group is finished.

**Comment by bradrydzewski** [comment]
Please open a thread at discourse.drone.io where we provide user support

**Comment by jamrizzi** [comment]
I did that, but my account was put on hold.
